### ISSUE TITLE
Duplicate CLI options

### ISSUE THREAD

**Issue by sjakobi** [Bodigrim/tasty-bench#3]
The `-j / --num-threads` and  `--csv` options appear twice in the options list:

```
Available options:
  -h,--help                Show this help text
  -p,--pattern PATTERN     Select only tests which satisfy a pattern or awk
                           expression
  -t,--timeout DURATION    Timeout for individual tests (suffixes: ms,s,m,h;
                           default: s)
  -l,--list-tests          Do not run the tests; just print their names
  -j,--num-threads NUMBER  Number of threads to use for tests execution
                           (default: # of cores/capabilities)
  --csv ARG                File to write results in CSV format. If specified,
                           suppresses console output
  -j,--num-threads NUMBER  Number of threads to use for tests execution
                           (default: # of cores/capabilities)
  -q,--quiet               Do not produce any output; indicate success only by
                           the exit code
  --hide-successes         Do not print tests that passed successfully
  --color never|always|auto
                           When to use colored output (default: auto)
  --ansi-tricks ARG        Enable various ANSI terminal tricks. Can be set to
                           'true' or 'false'. (default: true)
  --stdev ARG              Target relative standard deviation of measurements in
                           percents (5 by default). Large values correspond to
                           fast and loose benchmarks, and small ones to long and
                           precise. If it takes far too long, consider setting
                           --timeout, which will interrupt benchmarks,
                           potentially before reaching the target deviation.
  --csv ARG                File to write results in CSV format. If specified,
                           suppresses console output
```

**Comment by Bodigrim** [comment]
This has been accidentally fixed in f9691c5, I believe. At least I cannot reproduce with `master`.

**Comment by sjakobi** [comment]
Great! :)
